pt-BR/command_line.md

```diff
@@ -1,23 +1,25 @@
 **DO NOT READ THIS FILE ON GITHUB, GUIDES ARE PUBLISHED ON https://guides.rubyonrails.org.**
 
+Linhas de Comando do Rails
 The Rails Command Line
 ======================
 
-After reading this guide, you will know:
+Depois de ler este guia, você saberá:
 
-* How to create a Rails application.
-* How to generate models, controllers, database migrations, and unit tests.
-* How to start a development server.
+* Como criar uma aplicação Rails.
+* Como gerar modelos, controladores, migrações de banco de dados e testes de unidade.
```

Comment:
Acho que por ser uma coisa muito dos internos do Rails. Talvez não precisemos traduzir *models* e *controllers* só deixando em itálico.

Vou marcar a @erikacamposdesign que viu mais essa parte.

Reply:
```suggestion
* Como gerar *models*, *controllers, *migrations* de banco de dados e testes de unidade.
```